src/core/updaters_manual.go

```diff
@@ -0,0 +1,163 @@
+package core
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/trustwallet/assets-go-libs/pkg"
+)
```

Comment:
I don't think we need this updater anymore. It's no longer relevant due to us supporting many other coins now